src/main/java/org/symphonyoss/symphony/messageml/util/ShortID.java

```diff
@@ -0,0 +1,128 @@
+package org.symphonyoss.symphony.messageml.util;
+
+import java.security.SecureRandom;
+import java.util.Random;
+
+/**
+ * Short id generator. Url-friendly. Non-predictable. Cluster-compatible.
+ *
+ * <p></p>
+ * Inspired from <a href="https://github.com/dylang/shortid">shortid</a>
+ */
+public class ShortID {
```

Comment:
This class is used to generate shortid similar to one generated by the frontend by js: shortid.generate()